src/components/remotecontrol/remotecontrol.js

```diff
@@ -134,7 +134,7 @@ function imageUrl(item, options) {
 function updateNowPlayingInfo(context, state, serverId) {
     const item = state.NowPlayingItem;
     const displayName = item ? getNowPlayingNameHtml(item).replace('<br/>', ' - ') : '';
-    if (typeof item !== 'undefined') {
+    if (typeof item !== 'undefined' && item !== null) {
```

Comment:
Why so complicated?
```suggestion
    if (item) {
```

Reply:
I left a stricter check there because I don’t know why they checked for undefined only initially. It can be switched to the shorter version if there is no concern for unexpected behavior.

Reply:
Look at line above (this is legacy code): https://github.com/jellyfin/jellyfin-web/blob/7eff82d671d2df4c65988e229984af98a15189b8/src/components/remotecontrol/remotecontrol.js#L136
Stricter check was added #1056

Reply:
The commit that added it: https://github.com/jellyfin/jellyfin-web/commit/3ab01d471fa62a645bfd241484de9ebcfbf654aa#diff-41e838feabde38a10ddc99b4414d2379

Reply:
This is another old code (now it is inside that condition) and it also uses simple `(item)` (in fact, every test in function `updateNowPlayingInfo` is `(item)` until that PR):
https://github.com/jellyfin/jellyfin-web/blob/7eff82d671d2df4c65988e229984af98a15189b8/src/components/remotecontrol/remotecontrol.js#L231-L242
btw, `else` should be related to that `if`.

I see no harm in using `(item)` - `item` is supposed to be an object.